presto-docs/src/main/sphinx/release/release-0.213.rst

```diff
@@ -0,0 +1,99 @@
+  ``WHERE`` condition of the ``SELECT`` clause.
+* Do not allow setting invalid property values with :doc:`/sql/set-session`.
+* Disable stats calculator by default as it can cause a planning failure for
+  certain complex queries. It can be enabled with the ``experimental.enable-stats-calculator``
+  configuration property or the ``enable_stats_calculator`` session property.
+* Avoid making wild guesses when estimating filters for joins. Previously, if nothing
+  was known about the filter, a ``0.9`` coefficient was applied as a filter factor.
+  Now, if nothing is known about a filter, the estimate will be unknown. A ``0.9``
+  coefficient will be applied for all additional conjuncts if at least a single
+  conjunct can be reasonably estimated.
+* Improve inference of predicates for inner joins.
+* Improve ``EXPLAIN ANALYZE`` output by including *real* CPU time and scheduled time.
+* Include stats and cost estimates in textual plans created on query completion.
+* Remove transaction from explain plan for indexes.
+* Add ``max_drivers_per_task`` session property, allowing users to limit concurrency by
+  specifying a number lower than the system configured maximum. This can cause the
+  query to run slower and consume less resources.
+* Add ``join-max-broadcast-table-size`` configuration property and
+  ``join_max_broadcast_table_size`` session property to control the maximum estimated size
+  of a table that can be broadcast when using ``AUTOMATIC`` join distribution type (:issue:`11667`).
+* Add :func:`ST_MultiPoint` function to construct multi-point geometry from an array of points.
+* Add :func:`geometry_union` function to efficiently union arrays of geometries.
+* Add support for distributed spatial joins (:issue:`11072`).
+* Add experimental config option ``experimental.reserved-pool-enabled`` to disable the reserved memory pool.
+* Add ``targetResultSize`` query parameter to ``/v1/statement`` endpoint to control response data size.
+
+Server RPM Changes
+------------------
+
+* Allow running on a JVM from any vendor.
+
+Web UI Changes
+--------------
+
+* Remove legacy plan UI.
+* Add support for filtering queries by all error categories.
+* Add dialog to show errors refreshing data from coordinator.
+* Change worker thread list to not show thread stacks by default to improve page peformance.
+
+Hive Connector Changes
+----------------------
+
+* Fix LZO and LZOP decompression to work with certain data compressed by Hadoop.
+* Fix ORC writer validation percentage so that zero does not result in 100% validation.
+* Fix potential out-of-bounds read for ZSTD on corrupted input.
+* Stop assuming no distinct values when column null fraction statistic is less than ``1.0``.
+* Treat ``-1`` as an absent null count for compatibility with statistics written by
+  `Impala <https://issues.apache.org/jira/browse/IMPALA-7497>`_.
+* Preserve original exception for metastore network errors.
+* Preserve error messages from Avro and other deserializers.
```

Comment:
This could be "Preserve exceptions from Avro deserializer" because

- for Avro, full exceptions are preserved, because i provided a subclass of the serde
- for others, error messages are preserved.. except that apparently no other serde writes to `org.apache.hadoop.hive.serde2.AbstractSerDe#configErrors`, so adding "and error messages from other deserializers" isn't probably very useful for users